apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -9,7 +9,13 @@ import {
   studentLevelProgressPropType
 } from './sectionProgressRedux';
 import color from "../../util/color";
-import {progressStyles} from './VirtualizedSummaryView';
+import {progressStyles, ROW_HEIGHT, NAME_COLUMN_WIDTH, MAX_TABLE_SIZE} from './multiGridConstants';
+import i18n from '@cdo/locale';
+import SectionProgressNameCell from './SectionProgressNameCell';
+
+const PROGRESS_BUBBLE_WIDTH = 39;
```

Comment:
Good question. I view the constants file as a "shared constants" file, and since this variable is only used internally in this one component, I'll keep it here until it gets used elsewhere.